assets/js/components/Dashboard/AssociateAlertPage.tsx

```diff
@@ -0,0 +1,327 @@
+import React, { ComponentType, useState } from "react";
+import {
+  Button,
+  Container,
+  Row,
+  Col,
+  ButtonGroup,
+  Modal,
+  Form,
+} from "react-bootstrap";
+import { useNavigate } from "react-router-dom";
+import { useScreenplayContext } from "../../hooks/useScreenplayContext";
```

Comment:
We have some aliases now that makes these paths a bit easier to read. For example, you can say `Hooks/useScreenplayContext` here. You can find all aliases [here](https://github.com/mbta/screenplay/blob/59a41717febf6b907531ec18947b07791e66b3ea/assets/webpack.config.js#L60).